ivy_tests/test_ivy/test_functional/test_core/test_general.py

```diff
@@ -1399,6 +1399,19 @@ def test_inplace_increment(x_n_inc, tensor_fn, device, call):
     pytest.skip()
 
 
+@pytest.mark.parametrize("backend_str", [None,"numpy", "mxnet", "torch", "tensorflow", "jax"])
```

Comment:
Add single whitespace after None and also split it into two lines to prevent lint checks from failing